packages/reporter/src/controllers/WormholeReporterController.ts

```diff
@@ -0,0 +1,104 @@
+import axios, { AxiosInstance } from "axios"
```

Comment:
is `WormholeReporterController` used?

Reply:
`WormholeReporterController` has extra logic relevant to VAA, which is not required from `StandardReporterController`.
https://github.com/gnosis/hashi/blob/feat/reporter/packages/reporter/src/controllers/WormholeReporterController.ts#L63-L78